cpp/simple_crawler.cc

```diff
@@ -55,9 +46,9 @@ void Usage() {
               << "                                                            codes with commas.\n"
               << "\t[ (--print-redirects | -p) ]                              Nomen est omen.\n"
               << "\t[ (--timeout | -t) milliseconds ]                         Overall time we're willing to wait\n"
-              << "                                                            to download a page (default " + StringUtil::ToString(DEFAULT_TIMEOUT) + ").\n"
+              << "                                                            to download a page (default " + std::to_string(SimpleCrawler::DEFAULT_TIMEOUT) + ").\n"
               << "\t[ (--min-url-processing-time | -m) milliseconds ]         Min time between downloading 2 URLs\n"
```

Comment:
In English the plural of an acronym is written with an apostrophe before the "s".